security/security_profiles_operator/spo-release-notes.adoc

```diff
@@ -12,6 +12,24 @@ These release notes track the development of the Security Profiles Operator in {
 
 For an overview of the Security Profiles Operator, see xref:../../security/security_profiles_operator/spo-overview.adoc#[Security Profiles Operator Overview].
 
+[id="spo-release-notes-0-8-1"]
+== Security Profiles Operator 0.8.1
+
+The following advisory is available for the Security Profiles Operator 0.8.1:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5958[RHBA-2023:5958 - OpenShift Security Profiles Operator bug fix update]
+
+[id="spo-0-8-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, `SELinuxProfiles` did not inherit custom `SELinuxProfiles` from the same namespace. With this update, the issue has now been resolved and `SELinuxProfiles` are inherited from the same namespace as expected. (link:https://issues.redhat.com/browse/OCPBUGS-17164[*OCPBUGS-17164*])
```

Comment:
Could you please help to update all `SELinuxProfiles` to `SelinuxProfile`, or least the second one? It is case sentitive for the yaml file.
```
$ oc apply -f -<<EOF
> apiVersion: security-profiles-operator.x-k8s.io/v1alpha2
kind: SelinuxProfile
metadata:
  name: test-inherit-policy-selinuxprofile
  namespace: nginx-deploy
spec:
  inherit: 
  - kind: SELinuxProfiles
    name: nginx-secure
  allow:
    var_run_t:
      sock_file:
        - write
EOF
The SelinuxProfile "test-inherit-policy-selinuxprofile" is invalid: spec.inherit[0].kind: Unsupported value: "SELinuxProfiles": supported values: "System", "SelinuxProfile"
```

Reply:
Yes - I can change this to `SelinuxProfile` objects. :+1: